### ISSUE TITLE
VPAT — Homepage Does Not Have a Unique Page Title

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#4997]
As noted by an internal accessibility audit, the homepage view does not have a unique page title.
